Dockerfile

```
@@ -0,0 +1,53 @@
+# Use the official Node.js LTS image as the base image
+FROM node:lts
```

I'd suggest to pin the LTS version of node to avoid unexpected regressions.
```suggestion
FROM node:20
```